project/Settings.scala

```diff
@@ -109,7 +108,8 @@ object Settings {
     resolvers ++= Seq(
       Resolver.mavenLocal,
       Settings.Repositories.geosolutions,
-      Settings.Repositories.osgeo,
+      Settings.Repositories.osgeoSnapshots,
```

Comment:
Do we need the snapshots ?

Reply:
@echeipesh I think we don't; do you want me to remove them?

Reply:
Yeah, every split second counts in build times. Also it should be a big deal to find a snapshot version if we need to.

Reply:
💯